config/tablar.php

```diff
@@ -131,6 +131,28 @@
     */
     'display_alert' => false,
 
+    'alerts' => [
+        'default_position' => [
+            'top' => '10px',
+            'right' => '10px',
+        ],
+        'positions_by_layout' => [
```

Comment:
`'top' => '10px', 'right' => '10px'`

It's redundant code. Could you please reduce the redundancy?

Reply:
Hi @takielias,

Thank you for your feedback! The current implementation already attempts to reduce redundancy by using $defaultPosition as a fallback. Here's how it works:

The positions_by_layout configuration only defines overrides for specific layouts.
For layouts without specific overrides, $defaultPosition is applied:
php
Copia codice
$layoutPositions = config("tablar.alerts.positions_by_layout.$layout", $defaultPosition);
Finally, the individual position values (e.g., top, right, left) are extracted with ?? to ensure defaults are always applied.
This approach ensures minimal redundancy while allowing layout-specific overrides. However, if you prefer a different structure I’d be happy to adjust it. Please let me know your preference!